src/Surfnet/StepupRa/RaBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -99,8 +106,11 @@ public function authenticate(TokenInterface $token)
             $roles[] = 'ROLE_RA';
         }
 
+        $institutionConfigurationOptions = $this->institutionConfigurationOptionsService
+            ->getInstitutionConfigurationOptionsFor($identity->institution);
+
```

Comment:
Good point, make sure it's also logged as critical (as it would mean inconsistent state)...